models/contacts.js

```diff
@@ -35,7 +35,7 @@ const removeContact = async (contactId) => {
     }
     await fs.writeFile(contactsPath, JSON.stringify(updateContacts, null, 2));
   } catch (error) {
-    return error;
+    console.log(error);
```

Comment:
```suggestion
    console.log(error); // to jest wporządku 
    throw error;
```

Reply:
Jak robisz throw error tutaj to ten błąd zostanie przesłany do kolejnego bloku catch